src/features/filter.ts

```diff
@@ -1,10 +1,46 @@
-export const actions = { /* put action creators here */};
+type FilterTodosAction = { type: 'filter/FILTER', payload: string };
+const filterTodo = (value: string): FilterTodosAction => ({
+  type: 'filter/FILTER', payload: value,
+});
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type SelectTodosAction = { type: 'select/SELECT', payload: string };
+const selectTodo = (value: string): SelectTodosAction => ({
+  type: 'select/SELECT', payload: value,
+});
```

Comment:
Within one reducer you should use one prefix. You cannot have some actions start with `select` and other with `filter`